pkg/config/release.go

```diff
@@ -135,3 +143,16 @@ func GetAllConfigsFromSHA(releaseRepoPath, sha string, logger *logrus.Entry) (*R
 
 	return config, nil
 }
+
+func GetChangedClusterProfiles(path, baseSHA string) (ret []ClusterProfile, err error) {
+	diff, err := git(path, "diff-tree", "--diff-filter=d", baseSHA+":"+ClusterProfilesPath, "HEAD:"+ClusterProfilesPath)
+	if err != nil || diff == "\n" {
+		return
+	}
+	for _, l := range strings.Split(strings.TrimSpace(diff), "\n") {
+		var _s, hash, name string
+		fmt.Sscanf(l, ":%6s %6s %40s %40s %s\t%s\t", &_s, &_s, &_s, &hash, &_s, &name)
```

Comment:
oh jesus. At least can you try to add a comment for this line?

Reply:
We execute `git diff-tree` then parse its well-documented output to extract the hash and name from each line. What comment would you add that wouldn't be parroting what the code already says?

Reply:
It *would* be nice to ignore the values we don't care about (à lá c's `*` modifier) instead of using the dummy `_s` variable, but I don't think go supports that.

Reply:
Document your `_s` idea, it's not obvious

Reply:
So, it turns out the strings we are interested in appear in fixed offsets, so we can just index into the input string directly. I tried to think of every possible transformation and added them to the unit tests, the algorithm worked for all of them.